WSSiOS/WSSiOS/Resource/Constants/Strings/StringLiterals.swift

```diff
@@ -19,6 +23,18 @@ enum StringLiterals {
         static let userBirth = "USER_BIRTH"
     }
     
+    enum NotificationCenter {
```

Comment:
W2;
NotificationCenter 는 Swift에서 제공하는 기본 메커니즘과 이름이 동일하기 때문에 수정하는 게 어떨까요?
```suggestion
    enum FCMCenter {
```